themes/default/views/editor/objects/summary_html_ajax.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/* ----------------------------------------------------------------------
+ * views/editor/objects/summary_html_ajax.php :
```

Comment:
@kehh is this what we want the copyright block to look like for new templates we send upstream? (copied from other templates)

Reply:
Looks good just change the date